releasenotes/notes/gradient-based-adapt-vqe-aa4e89f44c5aa69e.yaml

```diff
@@ -0,0 +1,51 @@
+---
+features:
+  - |
+    Added Qiskit Terra's Gradient Functionality to calculate gradients in AdaptVQE. You can choose
+    the gradient method and input it while calling ``AdaptVQE``.
+    for example::
+      
+      from qiskit.providers.basicaer import BasicAer
+      from qiskit.utils import QuantumInstance
+      from qiskit.opflow.gradients import Gradient
+      from qiskit_nature.algorithms import VQEUCCFactory
+      from qiskit_nature.drivers import UnitsType
+      from qiskit_nature.drivers.second_quantization import PySCFDriver
+      from qiskit_nature.mappers.second_quantization import ParityMapper
+      from qiskit_nature.converters.second_quantization import QubitConverter
+      from qiskit_nature.problems.second_quantization import ElectronicStructureProblem
+      self.driver = PySCFDriver(
+            atom="H .0 .0 .0; H .0 .0 0.735", unit=UnitsType.ANGSTROM, basis="sto3g"
+            )
+      self.problem = ElectronicStructureProblem(self.driver)
+      self.qubit_converter = QubitConverter(ParityMapper())
+      solver = VQEUCCFactory(QuantumInstance(BasicAer.get_backend("statevector_simulator")))
+      grad = Gradient(grad_method="lin_comb", epsilon=1.0)
+      calc = AdaptVQE(self.qubit_converter, solver, gradient=grad)
+      res = calc.solve(self.problem)
+deprecates:
```

Comment:
```suggestion
deprecations:
```
This section is not correctly named is not appearing in release notes. I am not sure if thats why the section looks a bit odd or not. It maybe a prior note needs sorting too, not sure.

![image](https://user-images.githubusercontent.com/40241007/163001160-31bfa68b-d08d-4f96-92c7-1fb35912dbf5.png)